### PR TITLE
chore: update zombienet for test gh workflow

### LINKED ISSUE #548: out of date zombienet version in test workflow
## Bug Report

Dockerfile uses zombienet version 1.3.30 but test uses v1.3.18

https://github.com/paritytech/capi/blob/main/Dockerfile#L8

### PR DESCRIPTION
**Pull request by ryanleecode**
Resolves #548

### CODE REVIEW

File: providers/frame/Zombienet.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export interface ZombienetProviderProps {
 
 const defaultZombienetPaths: Record<string, string | undefined> = {
   darwin: "zombienet-macos",
-  linux: "zombienet-linux",
+  linux: "zombienet-linux-x64",
```
Comment (tjjfvi) [inline providers/frame/Zombienet.ts:15]: This should switch based on `Deno.build.arch`

Comment (ryanleecode) [reply providers/frame/Zombienet.ts:15]: aye

**Comment by ryanleecode** [comment]
```
Download http://localhost:4646/frame/wss/kusama-rpc.polkadot.io/@v0.9.37/mod.ts
pkg/prelude/bootstrap.js:1876
      throw error;
      ^

Error: EEXIST: file already exists, mkdir '/tmp/pkg'
    at Object.mkdirSync (node:fs:1349:3)
    at Object.mkdirSync (pkg/prelude/bootstrap.js:1642:33)
    at createDirRecursively (pkg/prelude/bootstrap.js:256:8)
    at createDirRecursively (pkg/prelude/bootstrap.js:255:5)
    at process.dlopen (pkg/prelude/bootstrap.js:2217:7)
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:11[89](https://github.com/paritytech/capi/actions/runs/4086663981/jobs/7046232800#step:9:90):18)
    at Module.load (node:internal/modules/cjs/loader:[98](https://github.com/paritytech/capi/actions/runs/4086663981/jobs/7046232800#step:9:99)1:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:[100](https://github.com/paritytech/capi/actions/runs/4086663981/jobs/7046232800#step:9:101)5:19)
    at Module.require (pkg/prelude/bootstrap.js:1855:31) {
  errno: -17,
  syscall: 'mkdir',
  code: 'EEXIST',
  path: '/tmp/pkg'
}
Download http://localhost:4646/frame/wss/westend-rpc.polkadot.io/@v0.9.37/mod.ts
Download http://localhost:4646/frame/wss/rococo-contracts-rpc.polkadot.io/@v0.9.370/mod.ts
Download http://localhost:4646/deps/std/bytes.ts
error: Uncaught Error: Zombienet exited without launching network
      throw new Error("Zombienet exited without launching network")
```

:monocle_face:

**Comment by ryanleecode** [comment]
@harrysolovay https://github.com/paritytech/capi/blame/571c9aff2d798ff658d5d36b0a57de59c28284c1/_tasks/download_frame_metadata.ts#L12 can this be removed? seems to be causing the error listed above.

**Comment by ryanleecode** [comment]
Nvm its just flaky

**Comment by tjjfvi** [comment]
What's the status of this?

**Comment by ryanleecode** [comment]
Tests are flaky. Sometime it passes and sometimes it doesnt

**Comment by ryanleecode** [comment]
looks to be working now

**Review by kratico — APPROVED** [review]
LGTM

the latest version is `v1.3.34`, perhaps we can update to that one

https://github.com/paritytech/zombienet/releases/tag/v1.3.34

**Review by harrysolovay — APPROVED** [review]

**Pull request merged**